### PR TITLE
Improved Details page:

### PR DESCRIPTION
**Pull request by danieleongari**
1) Removed redoundant titles from plots
2) Better naming for the tabs
3) Added the Process page
4) Added the Reported structure page

Problems to solve:
1) The process.png does not load in jupyter (only in panel) because pwd is detail
2) Qeq-CIF not displayed by Jmol
3) Jmol is not showing the to structures in the same page (the second one is loading forever)

### CODE REVIEW

File: detail/process.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+""" Plots the process details
+"""
+
+def print_process(process_node):
+    process_dict = process_node.get_dict()
+    html_text = '''
```
Comment (ltalirz) [inline detail/process.py]: markdown?

File: detail/process.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+<img src=\"process.png\">
```
Comment (ltalirz) [inline detail/process.py]: remove?

File: detail/main.ipynb [hunk 3]
```diff
@@ -113,20 +134,15 @@
     "\n",
     "tabs = pn.Tabs()\n",
     "tabs.extend([\n",
-    "    ('Structure', explorer.structure),\n",
-    "    ('Total energy', explorer.ener_plot),\n",
+    "    ('Process evaluation', explorer.process),\n",
+    "    ('Reported structure', explorer.structure_ref),\n",
+    "    ('DFT-optimization energy', explorer.ener_plot),\n",
+    "    ('DFT-optimized structure', explorer.structure_opt),\n",
```
Comment (ltalirz) [inline detail/main.ipynb:140]: could you comment one of the structures for the moment in order not to break things?

**Comment by ltalirz** [comment]
@danieleongari very sorry for the long delay - happy to merge after the minor fixes mentioned in the comments

**Comment by danieleongari** [comment]
Thanks, I fixed the mentioned errors, 
the new visualization was tested and works fine (`bokeh serve --show detail`)

**Comment by ltalirz** [comment]
thanks, Daniele!

**Pull request merged**